example/test/unit_test.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+// Consider `flutter test --no-test-assets` if assets are not required.
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  // This has the side effect of setting up a mock HTTP client.
+  // Disable this with HttpOverrides.global = null;
+
+  test('Can test fonts', () {
+    final styleFunc = GoogleFonts.asMap()['ABeeZee']!;
+    final expectedStyle = GoogleFonts.getFont('ABeeZee');
```

Comment:
The code snippet I shared was just one test case to show the issue I was facing. I also experimented this a bit, and I wonder if it makes more sense to put this in `setUp` or `setUpAll` to account for a bigger suite of tests? I tried this out, and it seems to be working too. So for example:

```dart
import 'package:flutter/material.dart';
import 'package:flutter_test/flutter_test.dart';
import 'package:google_fonts/google_fonts.dart';

void main() {
  TestWidgetsFlutterBinding.ensureInitialized();
  late TextStyle expectedStyle;

  setUpAll(() {
    expectedStyle = GoogleFonts.getFont('ABeeZee');
  });

  test('test fonts', () {
    final styleFunc = GoogleFonts.asMap()['ABeeZee']!;
    expect(styleFunc(), equals(expectedStyle));
  });
}
```

What do you think?

Reply:
I prefer showing this slightly more complex example to avoid people encoutering the same issue you did :thumbs_up